contracts/axelar-operators/src/contract.rs

```diff
@@ -85,7 +85,7 @@ impl AxelarOperators {
         let key = DataKey::Operators(operator);
```

Comment:
we're missing an extend ttl in this method as well, maybe create a task to revisit instance ttl for every contract?